src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/ReferenceAssemblies.cs

```diff
@@ -353,7 +353,9 @@ private async Task<ImmutableArray<MetadataReference>> ResolveCoreAsync(string la
                         }
                     }
 
-                    if (nearestFramework is object)
+                    // Ignore framework references for 'Any,Version=v0.0' until we know how NuGet is supposed to handle
+                    // them. https://github.com/dotnet/roslyn-sdk/issues/482
+                    if (nearestFramework is { IsAny: false })
```

Comment:
📝 The alternative to this is to only ignore the references if there is no reference assembly package specified.

Reply:
Can you file a bug on the nuget team and link it here?

Reply:
I filed #482 here and sent an email to the NuGet team to ask how we are supposed to resolve it.